persistence/sql/migratest/migration_test.go

```diff
@@ -72,323 +71,326 @@ func CompareWithFixture(t *testing.T, actual interface{}, prefix string, id stri
 	assert.NoError(t, s.SnapshotWithName(id, actualJSON))
 }
 
-func TestMigrations(t *testing.T) {
+func TestMigrations_SQLite(t *testing.T) {
+	t.Parallel()
 	sqlite, err := pop.NewConnection(&pop.ConnectionDetails{
 		URL: "sqlite3://" + filepath.Join(os.TempDir(), x.NewUUID().String()) + ".sql?_fk=true"})
 	require.NoError(t, err)
 	require.NoError(t, sqlite.Open())
 
-	connections := map[string]*pop.Connection{
-		"sqlite": sqlite,
+	testDatabase(t, "sqlite", sqlite)
+}
+
+func TestMigrations_Postgres(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping testing in short mode")
 	}
-	if !testing.Short() {
-		dockertest.Parallel([]func(){
-			func() {
-				connections["postgres"] = dockertest.ConnectToTestPostgreSQLPop(t)
-			},
-			func() {
-				connections["mysql"] = dockertest.ConnectToTestMySQLPop(t)
-			},
-			func() {
-				connections["cockroach"] = dockertest.ConnectToTestCockroachDBPop(t)
+	t.Parallel()
+	testDatabase(t, "postgres", dockertest.ConnectToTestPostgreSQLPop(t))
+}
+
+func TestMigrations_Mysql(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping testing in short mode")
+	}
+	t.Parallel()
+	testDatabase(t, "mysql", dockertest.ConnectToTestMySQLPop(t))
+}
+
+func TestMigrations_Cockroach(t *testing.T) {
```

Comment:
This setup is a lot easier to use. I've had to comment out parts of the old setup from time to time, if say the mysql migrations were failing, and I wanted to debug. Now, I can just click debug on the `TestMigrations_Mysql` case. WDYT?